srcpkgs/anbox-modules-git/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'anbox-modules-git'
+pkgname=anbox-modules-git
+version=0.0.0.20180908
+revision=1
+_commit=27fd47e11ef6eef93738f8f3df3e42c88975544e
+wrksrc="${pkgname%-git}-${_commit}"
+conf_files="/etc/modules-load.d/anbox.conf"
+short_desc="Anbox kernel modules"
```

Comment:
Maybe `Kernel modules necessary for the Anbox Android container runtime` or something more descriptive?

Reply:
Sounds good.